authors/apps/articles/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from ..authentication.models import User
+from datetime import datetime, timedelta
+
+from taggit.managers import TaggableManager
+
+
+# Create your models here.
+
+class Article(models.Model):
+
+
+    title = models.CharField(db_index= True, max_length = 255)
+    description = models.CharField(db_index=True, max_length = 255)
+    body = models.TextField()
+    created_at = models.DateField(auto_now_add=True)
+    published = models.BooleanField()
+    updated_at = models.DateTimeField(auto_now=True)
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    tags = TaggableManager(blank=True)
```

Comment:
In regards to this section, I think it would be best if we aligned our code with what the documentation entails so that it's easier to have other team members use it with ease .

So my suggestion would be to change this to 

`"tagList":`